src/odemis/driver/autoscript_client.py

```diff
@@ -0,0 +1,2065 @@
+            self.parent.get_scan_rotation(self.channel),
+            rotation_info["range"],
+            unit=rotation_info["unit"],
+            setter=self._setRotation)
+
+        # horizontal field of view
+        fov_info = self.parent.field_of_view_info(self.channel)
+        fov = self.parent.get_field_of_view(self.channel)
+        self.horizontalFoV = model.FloatContinuous(
+            fov,
+            unit=fov_info["unit"],
+            range=fov_info["range"],
+            setter=self._setHorizontalFoV)
+        self.horizontalFoV.subscribe(self._onHorizontalFoV)
+
+        mag = self._hfw_nomag / fov
+        mag_range_max = self._hfw_nomag / fov_info["range"][0]
+        mag_range_min = self._hfw_nomag / fov_info["range"][1]
+        self.magnification = model.FloatContinuous(mag, unit="",
+                                                   range=(mag_range_min, mag_range_max),
+                                                   readonly=True)
+        # To provide some rough idea of the step size when changing focus
+        # Depends on the pixelSize, so will be updated whenever the HFW changes
+        self.depthOfField = model.FloatContinuous(1e-6, range=(0, 1e3),
+                                                  unit="m", readonly=True)
+        self._updateDepthOfField()
+
+        if has_detector:
+            rng = DETECTOR_RNG
+            self._shape = (rng[1][0], rng[1][1])
+            # pixelSize is the same as MD_PIXEL_SIZE, with scale == 1
+            # == smallest size/ between two different ebeam positions
+            pxs = (fov / self._shape[0],
+                   fov / self._shape[0])
+            # pixelsize is inferred indirectly via resolution and fov
+            self.pixelSize = model.VigilantAttribute(pxs, unit="m", readonly=True)
+
+            # scanning resolution
+            resolution = self.parent.get_resolution(self.channel)
+            res_choices = set(RESOLUTIONS)
+            self.resolution = model.VAEnumerated(resolution, res_choices, unit="px" , setter=self._setResolution)
+
+            # (float, float) as a ratio => how big is a pixel, compared to pixelSize
+            # it basically works the same as binning, but can be float.
+            # Defined as the scale to match the allowed resolutions, with pixels
+            # always square (ie, scale is always the same in X and Y).
+            scale = (self._shape[0] / resolution[0],) * 2
+            scale_choices = set((self._shape[0] / r[0],) * 2 for r in res_choices)
+            self.scale = model.VAEnumerated(scale, scale_choices, unit="", setter=self._setScale)
+            self.scale.subscribe(self._onScale, init=True)  # to update metadata
+
+            # Just to make some code happy
+            self.translation = model.TupleContinuous((0, 0), range=[(-512, -512), (512, 512)], unit="px", readonly=True)
+
+        # not yet implemented:
+        # beam on
+        # beam blank
```

Comment:
You have the code for `.blanker`, and even code that relies on it. It should be pretty straightforward to add the VA. Or is there something that doesn't work with the blanker? If so, please remove all code related to .blanker then.